### DISCUSSION TITLE
After upgrading Ceph from Pacific to Reef, the performance has degraded

### DISCUSSION THREAD

**Discussion by microyahoo** [rook/rook#14553] [Q&A]
Under the same test conditions, it was found that the S3 performance decreased after upgrading Ceph. Additionally, I noticed that the `osd_op_queue` switched from `wpq` in Pacific to `mclock_scheduler` in Reef. I'm not sure if this is causing the issue. 

I tested the following scenarios: 
1. Switching `osd_op_queue` back to `wpq`; 
2. Using `mclock_scheduler` and setting the following parameters. 
```
osd                                     basic     osd_mclock_iops_capacity_threshold_ssd   300000.000000
osd                                     basic     osd_mclock_max_capacity_iops_ssd         300000.000000
osd                                     basic     osd_mclock_max_sequential_bandwidth_ssd  5033164800
osd                                     advanced  osd_mclock_profile                       high_client_ops
osd                                     advanced  osd_op_queue                             mclock_scheduler     *
```
However, I did not observe any performance improvement. Could you please give me some suggestions? Thanks in advance. 
@travisn @satoru-takeuchi @BlaineEXE 

before upgrade:
![20240809-225732](https://github.com/user-attachments/assets/859262c2-b0f3-4605-b3b8-356e0e4385cf)

after upgrade:
![20240809-225725](https://github.com/user-attachments/assets/51241302-5dd9-45a3-bf45-9272522a7fb9)
![20240809-225709](https://github.com/user-attachments/assets/78478ed3-ceac-4e2a-ba7a-f15f94747671)

**Comment by travisn** [comment]
I'd recommend opening a [Ceph tracker](https://tracker.ceph.com/) for the RGW team to take a look.

**Reply by microyahoo** [reply]
It seems this is related to the network. The bond degraded, and after restoring the bond, the performance improved.
